openapi-maven-plugin/src/main/java/io/github/kbuntrock/model/Tag.java

```diff
@@ -58,4 +66,13 @@ public String toString() {
 			", endpoints=" + endpoints.stream().map(Endpoint::toString).collect(Collectors.joining(", ")) +
 			'}';
 	}
+
+	@Override
+	public int compareTo(final Tag o) {
+		return Comparator
+			.comparing((Tag t) -> t.computedName, nullsLast(String::compareTo))
+			.thenComparing(t -> t.name, nullsLast(String::compareTo))
+			.thenComparing(t -> t.clazz.getCanonicalName())
```

Comment:
I think it makes no sense to sort by clazz name

Reply:
in case of name collision, it gives a more deterministic output

Reply:
But the class names are the same